lib/routes/hlju/news.ts

```diff
@@ -0,0 +1,181 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/news/:category?',
+    categories: ['university'],
+    example: '/hlju/news/hdyw',
+    parameters: {
+        category: {
+            description: '新闻分类，默认为黑大要闻',
+            options: [
+                { value: 'hdyw', label: '黑大要闻' },
+                { value: 'jjxy', label: '菁菁校园' },
+                { value: 'rwfc', label: '人物风采' },
+                { value: 'xwdt', label: '新闻动态' },
+                { value: 'jxky', label: '教学科研' },
+                { value: 'xyjw', label: '学院经纬' },
+                { value: 'jlhz', label: '交流合作' },
+                { value: 'cxcy', label: '创新创业' },
+            ],
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['hdxw.hlju.edu.cn/:category.htm', 'hdxw.hlju.edu.cn/'],
+            target: '/news/:category',
+        },
+    ],
+    name: '新闻网',
+    maintainers: ['nczitzk'],
+    handler,
+};
+
+async function handler(ctx) {
+    const category = ctx.req.param('category') ?? 'hdyw';
+    const baseUrl = 'https://hdxw.hlju.edu.cn';
+
+    // 栏目名称映射
+    const categoryMap = {
+        hdyw: '黑大要闻',
+        jjxy: '菁菁校园',
+        rwfc: '人物风采',
+        xwdt: '新闻动态',
+        jxky: '教学科研',
+        xyjw: '学院经纬',
+        jlhz: '交流合作',
+        cxcy: '创新创业',
+    };
+
+    const categoryName = categoryMap[category] || '黑大要闻';
+    const listUrl = `${baseUrl}/${category}.htm`;
+
+    const response = await ofetch(listUrl);
+
+    const $ = load(response);
+
+    // 提取新闻列表项 - 基于实际页面结构
+    const list: Array<{
+        title: string;
+        link: string;
+        pubDate?: Date;
+    }> = [];
+
+    // 查找所有新闻链接 - 匹配 info/ 路径的链接
+    $('a[href*="info/"]').each((_, element) => {
+        const item = $(element);
+        const link = item.attr('href');
+        const title = item.text().trim();
+
+        if (!title || !link || title.length < 5) {
+            return;
+        }
+
+        // 查找日期信息 - 在同一行或附近
+        const parent = item.parent();
+        const dateMatch = parent.text().match(/(\d{4})\/(\d{2})\/(\d{2})/);
+
+        list.push({
```

Comment:
Use [map](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead of [push](https://www.measurethat.net/Benchmarks/Show/11532/0/push-vs-map)